### PR TITLE
Add Subtitle component

### PR DESCRIPTION
**Pull request by evulins**
Add Problem Subtitle Header

### CODE REVIEW

File: src/components/atoms/ProblemSubtitle/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { screen } from "@testing-library/react";
+import { testRenderer } from "../../../setupTests";
+
+import ProblemSubtitle from "./index";
```
Comment (franiakat) [inline src/components/atoms/ProblemSubtitle/index.test.tsx]:
```suggestion
import { screen } from "@testing-library/react";

import { testRenderer } from "../../../setupTests";
import ProblemSubtitle from "./index";
```

File: src/components/atoms/ProblemSubtitle/interface.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export interface ProblemSubtitleProps {
+  subtitle: string;
+  color?: string;
+}
+
+export interface SubtitleHeaderStyleProps {
+  color?: string;
+}
```
Comment (franiakat) [inline src/components/atoms/ProblemSubtitle/interface.ts]:
```suggestion
export interface ProblemSubtitleProps extends SubtitleHeaderStyleProps {
  subtitle: string;
}

export interface SubtitleHeaderStyleProps {
  color?: string;
}
```

File: src/components/atoms/ProblemSubtitle/styles.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+
+import { SubtitleHeaderStyleProps } from "./interface";
+import Wrapper from "../Wrapper";
+import { minDevice } from "../../../assets/styles/breakpoints";
+import { blue200, placeholder } from "../../../assets/styles/colors";
+
+const SubtitleWrapper = styled(Wrapper)`
+  color: ${placeholder};
```
Comment (franiakat) [inline src/components/atoms/ProblemSubtitle/styles.tsx]: I'd rename this color, similar to what @bazelajoanna did with blue200 and so on). Now seeing its usage we see the name should be different, as it looks a bit odd here. ;)

**Review by franiakat — COMMENTED** [review]
Looking good! 😎 Just some minor details to consider.

**Review by bazelajoanna — COMMENTED** [review]
🥇 👩‍🎓

File: src/components/atoms/ProblemSubtitle/styles.tsx (hunk 3 above)
Comment (evulins) [reply src/components/atoms/ProblemSubtitle/styles.tsx]: Done :)

**Review by franiakat — APPROVED** [review]
sqash

**Pull request merged**